src/pjpersist/serializers.py

```diff
@@ -28,11 +28,11 @@ def read(self, state):
         return datetime.datetime.strptime(state['value'], self.fmt).date()
 
     def can_write(self, obj):
-        return isinstance(obj, datetime.date)
+        return type(obj) is datetime.date
 
     def write(self, obj):
         return {'_py_type': 'datetime.date',
-                'value': obj.strftime(self.fmt)}
```

Comment:
Eventually put that number right beside the `fmt` declaration? Those go hand in hand.

Reply:
name it something like `fmtLength`

Reply:
Good point, I think I will also add something like `FMT_DATE_LENGTH` to `serialize.py`